script.video.nfl.gamepass/addon.xml

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+  <addon id="script.video.nfl.gamepass"
+    name="NFL Game Pass"
+    version="2022.05.14"
+    provider-name="Alexqw,divingmule,BaumSchorle,eriksoderblom,kaileu,emilsvennesson,baardisk,Duke">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.m3u8" version="0.5.4+matrix.2"/>
+    <import addon="script.module.requests" version="2.22.0+matrix.1"/>
+    <import addon="inputstream.adaptive" version="19.0"/>
+  </requires>
+  <extension point="xbmc.python.script" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">NFL Game Pass</summary>
+    <description lang="en_GB">Watch NFL Game Pass streams.</description>
+    <platform>all</platform>
+    <language>en</language>
+    <license>GPL-2.0-only</license>
+    <source>https://github.com/pigskin/kodi-gamepass</source>
+    <forum></forum>
```

Comment:
can you please fill the forum support thread url if there's one, otherwise just remove this from the addon.xml?

Reply:
Removed the empty item, there is no forum support URL (yet)